contents/product-engineers/activation-metrics.mdx

```diff
@@ -0,0 +1,286 @@
+---
+date: 2024-11-13
+title: How we found our activation metric (and how you can too)
+author:
+  - raquel-m-smith
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/hog_ql.png
+featuredImageType: full
+category: Engineering
+tags:
+  - Product analytics
+  - Product engineers
+  - Growth
+  - Inside PostHog
+crosspost:
+  - Blog
+---
+
+People are signing up for your product. Great! But before you start planning your IPO, you need to make sure they actually stick around. This is the goal of **activation**.
+
+The trouble with activation is that it is hard to define. Many say it is when a user reaches the "aha" moment of your product, but this is vague and different for everyone.
+
+For example, we have 8+ products and each has a different set of activation metrics from launching a survey to inviting a teammate to creating two or more dashboards. What made finding activation metrics tricky was that each product had 10+ events that **could** be part of its activation metric.
+
+This means we spent a lot of time discovering our products' activation metrics. We're sharing our process here to help you do the same.
+
+But before we jump in... what is activation _really_?
+
+## What is an activation metric?
+
+Activation metrics are a set of actions a new user performs that are correlated with greater retention.
+
+Focusing on retention solves the trickiest part of defining an activation metric: it could be _anything_. Does a pageview count as activation? How about inviting a teammate? Creating their first object? Depending on your product, any or all of these could be an activation metric. It is **retention** that makes it clear.
```

Comment:
I think pageview isnt a good example. Not sure what could be better though. 
```suggestion
Focusing on **retention** is the trickiest part of defining an activation metric: it could be _anything_. Does spending 10 or more minutes in your app count as activation? How about inviting a teammate? Creating their first object? Depending on your product, any or all of these could play a part in retention
```

Reply:
Changed the example to "completing onboarding" but this wording changes the meaning. I am trying to say that activation could be anything, but you want to find the metrics that have the biggest impact on retention and then work backwards if that makes sense.